docs/secret_management.md

```diff
@@ -219,6 +219,24 @@ spec:
           mountPath: /etc/secret-volume
 ```
 
+The Datadog Agent also includes a script that can be used to read secrets from files mounted from Kubernetes secrets, or directly from Kubernetes secrets. This script can be used by setting `DD_SECRET_BACKEND_COMMAND` to `/readsecret_multiple_providers.sh`. A simple example of how to configure the DatadogAgent spec is provided below. For more details, see the [documentation][2].
+
+```yaml
+apiVersion: datadoghq.com/v1alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  credentials:
+    apiKey: ENC[k8s_secret@default/test-secret/api_key]
+    appKey: ENC[k8s_secret@default/test-secret/app_key]
+    useSecretBackend: true
```

Comment:
instead of documenting how to add an envvar.
should we add an option in the `credentials` section to select which secret backend script they want to use?

Reply:
I was thinking of saving this for v2, i.e. creating a SecretBackend section (and removing `credentials.useSecretBackend`). I could add on to the existing `credentials` section for now if you think it's worth doing?